SevenRed/MainForm.cs

```diff
@@ -0,0 +1,198 @@
+﻿using SevenRedLibrary;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
+
+namespace SevenRed
+{
+    public partial class MainForm : Form
+    {
+        Point lastPoint;
+
+        List<Card> WhiteDeck;
+        List<Card> BlackDeck;
+        
+        public MainForm()
+        {
+            InitializeComponent();
+        }
+
+        /// <summary>
+        /// Closes the application
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ClouseButton_Click(object sender, EventArgs e)
+        {
+            Close();
+        }
+
+        /// <summary>
+        /// Movement of window
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void MainForm_MouseMove(object sender, MouseEventArgs e)
+        {
+            if (e.Button == MouseButtons.Left)
+            {
+                Left += e.X - lastPoint.X;
+                Top += e.Y - lastPoint.Y;
+            }
+        }
+
+        /// <summary>
+        /// Find point of movement
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void MainForm_MouseDown(object sender, MouseEventArgs e)
+        {
+            lastPoint = new Point(e.X, e.Y);
+        }
+
+        /// <summary>
+        /// Add Cards to TextBoxes and check of correct data
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void AddCards_Click(object sender, EventArgs e)
+        {
+            List<string> whiteDeck = WhiteDeckText.Lines.ToList();
+            List<string> blackDeck = BlackDeckText.Lines.ToList();
+
+            try
+            {
+                if (whiteDeck.Count <= 8 && blackDeck.Count <= 8)
+                {
+                    WhiteDeck = Service.ConvertListStringToListCard(whiteDeck);
+                    BlackDeck = Service.ConvertListStringToListCard(blackDeck);
+                    
+                    List<TextBox> WhiteSlot = new List<TextBox>(7) { whiteCard1, whiteCard2, whiteCard3, whiteCard4, whiteCard5, whiteCard6, whiteCard7 };
```

Comment:
You're using constructor of class List with capacity - my respect to you for that